openapi.yaml

```diff
@@ -794,11 +794,11 @@ components:
           example: 0
         totalDebit:
           description: Accumulated Batch debit totals within the file.
-          type: integer
+          type: long
```

Comment:
I think we need to specify `format: int64` and keep `type: integer`. 

See: https://swagger.io/docs/specification/data-models/data-types/#numbers